### ISSUE TITLE
Specify "dev" for minimum-stability in README

### ISSUE THREAD

**Issue by bighappyface** [Jeff-Cardwell/adug_from_scratch_build#1]
It helps when attempting to install a contrib module later. The `composer.json` file example has the value but the interactive composer init doesn't provide the example.

**Comment by jeff-cardwell** [comment]
I agree that it is confusing the way it reads.  The section where I show the answers used follows the explanation of the questions.  I'll reverse their order and mention that explanation follows.  That sound good?

**Comment by bighappyface** [comment]
Your call :) Feel free to close the PR/Issue at will.
